src/index.js

```diff
@@ -0,0 +1,34 @@
+const BigNumber = require("bignumber.js");
+
+const { getInputOutputFiles, parseCsvFile, writeJson } = require("./utils");
+const GNO_ADDRESS = "0x6810e776880C02933D47DB1b9fc05908e5386b96";
+
+function toPayment(leaderBoardItem) {
+  const { trader, gno_estimation: gnoEstimation } = leaderBoardItem;
+
+  return {
+    amount: gnoEstimation,
+    tokenAddress: GNO_ADDRESS,
+    receiver: trader,
+  };
+}
+
+async function main() {
+  // Parse CSV
+  const [inputFile, outputFile] = getInputOutputFiles();
+  const results = await parseCsvFile(inputFile);
+
+  // Transform into payment object
+  const payments = results
+    .map(toPayment)
+    // Remove payment with amount zero
+    .filter((payment) => {
+      const amount = new BigNumber(payment.amount);
+      return !amount.isZero();
```

Comment:
What is the threshold for "isZero". Like how much precision do we bother with? Note that GNO has 18 decimals so our "airdrop" will also lose values below 10^{-18}.

Reply:
the query returns 0 if it's smaller than 0.0001 already
I can add it also the logic here, but I'm not sure I should decide on the minimum amount to send.

even a small amount of GNO today can be a great value tomorrow, thanks to our hard work, of course